e2etest/pkg/frr/container/container.go

```diff
@@ -69,6 +69,9 @@ func Start(name string, nc config.NeighborConfig, rc config.RouterConfig, networ
 		}
 	}
 
+	// setting routerid after calculating ips
+	frr.RouterConfig.RouterID = frr.Ipv4
```

Comment:
routerID is constant has nothing to do with interface IPs i.e u don't have to wait to intf IPs to set it can be any 32bits value in ipv4 dotted fmt

Reply:
isn't that a typical way to generate / set the ID though?

Reply:
True although it is commonly set to an IPv4 address. Makes it easier to identify routers and prevent duplicates.

Reply:
typical way in router land was to use loopback ip which some random ip 10.10.10.10 something like that again its just uint32 used in the bgp open message

Reply:
@fedepaol I checked slack and I saw the crash with error msg not sure what led to that but I found hard to believe this will crash bgpd we ran into bad bgp id alot when we were using views but that never crashed bgpd the session just never get to established

Reply:
I had a private chat with @ton31337 where he was saying that's the cause of the error, looking at the extended logs I put in the frr issue.

Reply:
I'll merge this, it won't hurt anyhow and it's test only.